src/vm/ceeload.cpp

```diff
@@ -6482,6 +6482,63 @@ void Module::NotifyDebuggerUnload(AppDomain *pDomain)
 }
 
 #if !defined(CROSSGEN_COMPILE)
+using GetTokenForVTableEntry_t = mdToken(STDMETHODCALLTYPE*)(HMODULE module, BYTE**ppVTEntry);
+
+HMODULE GetIJWHostForModule(Module* module)
```

Comment:
I am going to assume this function isn't used outside this compilation unit. Please mark it `static` if it is.